Style: Added arrow to grouped results

AlexanderBlanchardAC opened the pull request:
Added a downwards arrow next to available formats when a search returns grouped results in mobile view in order to increase how intuitive it is to access these results. Also, once results are displayed, this arrow provides the option to hide this section again. 
TEST PLAN:
1)Ensure that your search will contain grouped works. 
2)Carry out your search in a mobile view. 
3)Notice that results without grouped works are displayed as normal, but those with grouped works do not display 'place hold', 'on shelf' etc. 
4)Click the blue link that show the types of format available e.g. 'Books,' 'Visual Materials.'
5)Note that you should now be able to see the different format options and 'place hold,' 'on shelf' etc. Also note that you cannot hide these results again. 
6)Apply patch. 
7)Again carry out a search that includes some grouped results. 
8)Note that next to the available formats, there is now a small downward arrow. 
9)Click the arrow to display the results and the possible actions for each item. 
10)Click the arrow again and note that these results are now once again hidden. 
11)Switch to a desktop view and note that the grouped results are all displayed automatically and the format options and arrow are no longer visible.

AlexanderBlanchardAC committed:
Style: Added arrow to grouped results

mdnoble73 committed:
Merge branch '23.08.11' into 23.09.00

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
update file permission

mdnoble73 committed:
Updates for installing CLAM AV

mdnoble73 committed:
Antivirus updates

mdnoble73 committed:
Antivirus updates

mdnoble73 committed:
Add note in library settings

HitScan committed:
Use supplied db host when creating new database

Signed-off-by: Jason Boyer <JBoyer@equinoxOLI.org>

mdnoble73 committed:
Fix secondary phone number

mdnoble73 committed:
Fixes for detecting infected files

mdnoble73 committed:
Merge pull request #1413 from EquinoxOpenLibraryInitiative/jboyer/23.10.00/remote_db

Use supplied db host when creating new database

mdnoble73 committed:
Prevent patrons from cancelling ILL Holds loaded from Koha

mdnoble73 committed:
Merge branch '23.08.11' into 23.09.00

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Allow files to not be deleted when moving files

mdnoble73 committed:
Handle cases when authentication tokens time out while loading records for display within a list

mdnoble73 committed:
Add additional protection when we can't load records from EBSCOhost when showing list results

mdnoble73 committed:
Antivirus scan updates

mdnoble73 committed:
Make email not a required field

mdnoble73 committed:
Backwards compatible PHP functions

mdnoble73 committed:
Remove old loading of sessions

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
When starting or stopping Masquerade, create a new session

mdnoble73 committed:
If a MARC record does not include an 008 field, do not show an error and instead return a default language of English for the record.

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

catsoup11789 committed:
add php-ldap to installers

catsoup11789 committed:
comment out concat bits

catsoup11789 committed:
Update upgrade_23.09.00.sh

mdnoble73 committed:
Extract and display MARC Holdings from Sierra

mdnoble73 committed:
Merge pull request #1414 from catsoup11789/23.09.00

add php-ldap to installers

catsoup11789 committed:
Create upgrade_debian_23.09.00.sh

catsoup11789 committed:
make upgrade_debian_23.09.00.sh executable

catsoup11789 committed:
make upgrade_debian_23.09.00.sh executable

catsoup11789 committed:
Merge branch 'mdnoble73:23.09.00' into 23.09.00

mdnoble73 committed:
Fix updating cron for clamscan

mdnoble73 committed:
Merge branch '23.09.00' of https://github.com/mdnoble73/aspen-discovery into 23.09.00

K-Alette committed:
Update OverDriveDriver.php

mdnoble73 committed:
Merge pull request #1416 from K-Alette/23.09.00

Update OverDriveDriver.php

mdnoble73 committed:
Merge pull request #1415 from catsoup11789/23.09.00

Create upgrade_debian_23.09.00.sh

K-Alette committed:
Update OverDriveDriver.php

mdnoble73 committed:
Merge pull request #1417 from K-Alette/23.09.00

Update OverDriveDriver.php

mdnoble73 committed:
Update runScheduledUpdate.php

mdnoble73 committed:
Update runScheduledUpdate.php

mdnoble73 committed:
Update runScheduledUpdate.php

mdnoble73 committed:
Update runScheduledUpdate.php

mdnoble73 committed:
update file permission

catsoup11789 committed:
check circ_rules for auto_renew

catsoup11789 committed:
Create 23.09.01.MD

mdnoble73 committed:
Merge pull request #1419 from catsoup11789/23.09.01

check circ_rules for auto_renew

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

K-Alette committed:
Explore More Icon Images

Adds ability to upload images for the explore more icons (settings in themes)
Updated release notes

mdnoble73 committed:
Fix display of date/time picker on timestamp fields.

mdnoble73 committed:
Merge pull request #1420 from K-Alette/23.10.00

Explore More Icon Images

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
update file permission

mdnoble73 committed:
Update 23.09.01.MD

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

Commit:
Removed dependency on ILS-DI for user authentication for Koha versions >= 22.11.10 and above.

catsoup11789 committed:
fix for if no renewals are allowed

mdnoble73 committed:
Merge pull request #1422 from catsoup11789/23.09.01

fix for if no renewals are allowed

catsoup11789 committed:
Update AJAX.php

catsoup11789 committed:
Merge branch 'mdnoble73:23.09.01' into 23.09.01

mdnoble73 committed:
Merge pull request #1423 from catsoup11789/23.09.01

fix error message for scheduling update

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

catsoup11789 committed:
add uid to overpayWarning

catsoup11789 committed:
Merge branch '23.09.01' of https://github.com/catsoup11789/aspen-discovery into 23.09.01

catsoup11789 committed:
Merge branch 'mdnoble73:23.09.01' into 23.09.01

catsoup11789 committed:
Update 23.09.01.MD

catsoup11789 committed:
Merge branch '23.09.01' of https://github.com/catsoup11789/aspen-discovery into 23.09.01

mdnoble73 committed:
Merge pull request #1424 from catsoup11789/23.09.01

add uid to overpayWarning

mdnoble73 committed:
Update index.php

mdnoble73 committed:
Merge branch '23.09.01' of https://github.com/mdnoble73/aspen-discovery into 23.09.01

mdnoble73 committed:
MARC Holding idnexing and  display updates

catsoup11789 committed:
force int on outstanding_amt

mdnoble73 committed:
Use the PatronSelfModificationBorrowerMandatoryField when available rather than PatronSelfRegistrationBorrowerMandatoryField to determine required fields for patron updates.

catsoup11789 committed:
Merge branch 'mdnoble73:23.09.01' into 23.09.01

mdnoble73 committed:
Update the system files while updating Aspen on Debian systems

mdnoble73 committed:
Merge pull request #1425 from catsoup11789/23.09.01

force int on outstanding_amt

mdnoble73 committed:
Updates for comparing how much a patron is paying

mdnoble73 committed:
Fix logins for new users

catsoup11789 committed:
Update LDAPAuthentication.php

AlexanderBlanchardAC committed:
Disable Cookie Consent

mdnoble73 committed:
Evolve login debugging

mdnoble73 committed:
patron login

mdnoble73 committed:
Allow pickup location to be updated for holds that ready for pickup

catsoup11789 committed:
ACI updates

mdnoble73 committed:
Merge pull request #1426 from catsoup11789/23.09.01

add session_destroy()

catsoup11789 committed:
Update LDAPAuthentication.php

mdnoble73 committed:
Merge pull request #1431 from catsoup11789/23.09.01

Update LDAPAuthentication.php

mdnoble73 committed:
Additional safeguards to null barcodes, usernames, and borrower numbers are handled appropriately if detected within Koha

mdnoble73 committed:
MARC Holdings scoping

mdnoble73 committed:
Merge pull request #1430 from catsoup11789/23.10.00

ACI updates

mdnoble73 committed:
Sierra Indexing updates

Index secondary call number prestamp
Extrract item url and description from Sierra

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

catsoup11789 committed:
Update Koha.php

catsoup11789 committed:
Update Koha.php

mdnoble73 committed:
Merge pull request #1432 from catsoup11789/23.09.01

Update Koha.php

mdnoble73 committed:
Updates for Antivirus checking

mdnoble73 committed:
Antivirus monitoring updates

mdnoble73 committed:
Antivirus monitoring

mdnoble73 committed:
Updates for masquerading with a patron barcode.

mdnoble73 committed:
Correct issue masquerading as users that have never logged in before

mdnoble73 committed:
Antivirus monitoring

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Antivirus monitoring

mdnoble73 committed:
Antivirus monitoring

mdnoble73 committed:
Antivirus monitoriing

catsoup11789 committed:
LiDA bug fixes

mdnoble73 committed:
Correct issues that prevented masquerade mode from working properly

mdnoble73 committed:
Update upgrade_23.09.01.sh

mdnoble73 committed:
Merge branch '23.09.01' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Merge pull request #1421 from thekesolutions/authentication_by_api

Removed dependency on ILS-DI for user authentication for Koha versions >= 22.11.10 and above.

mdnoble73 committed:
Utilize new Rest API to validate patron accounts rather than ILSDI.

mdnoble73 committed:
Merge pull request #1428 from PTFS-Europe/disable-cookie-consent

Disable Cookie Consent

mdnoble73 committed:
Make Koha usernames and barcodes case insensitive when validating on login

K-Alette committed:
Num Entries & Contact Info/My Preferences

Fix for honoring number of entries to show for facets
Fix for updating wrong contact information and preferences
Updated release notes

mdnoble73 committed:
Merge pull request #1433 from K-Alette/23.10.00

Num Entries & Contact Info/My Preferences

mdnoble73 committed:
Merge branch '23.09.02' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

catsoup11789 committed:
autorenew messaging updates

catsoup11789 committed:
Update Koha.php

mdnoble73 committed:
Merge pull request #1434 from catsoup11789/23.09.03

autorenew messaging updates

mdnoble73 committed:
Merge branch '23.09.03' into 23.10.00

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

catsoup11789 committed:
initial add sys messages to app

catsoup11789 committed:
add isValidForDisplayInApp

catsoup11789 committed:
add APIs for getting sys messages & dismissing

catsoup11789 committed:
add initial context for system messages

catsoup11789 committed:
add in-app api calls for sys messages

catsoup11789 committed:
add message id

catsoup11789 committed:
update Forgot Barcode note

mdnoble73 committed:
Merge pull request #1435 from catsoup11789/23.10.00

initial code for system messages in LiDA

catsoup11789 committed:
Update SystemAPI.php

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

catsoup11789 committed:
aci speedpay updates

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

catsoup11789 committed:
disable button while processing

catsoup11789 committed:
Update Fines.php

mdnoble73 committed:
Optional Updates

mdnoble73 committed:
Merge branch '23.10.00' of https://github.com/mdnoble73/aspen-discovery into 23.10.00

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

catsoup11789 committed:
cleanup aci form

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

mdnoble73 committed:
Misc updates

catsoup11789 committed:
Update initialContext.js

mdnoble73 committed:
Merge pull request #1436 from catsoup11789/23.10.00

Update getSystemMessages in SystemAPI.php

mdnoble73 committed:
Merge branch '23.09.03' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Update 23.10.00.php

mdnoble73 committed:
Update Interface.php

catsoup11789 committed:
pre-load system messages

catsoup11789 committed:
add systemMessages context consumer

catsoup11789 committed:
add symphony note to course reserves

catsoup11789 committed:
Update library.js

catsoup11789 committed:
prevent early re-fetch

catsoup11789 committed:
add system messages to screens

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

catsoup11789 committed:
cleanup aci styles

K-Alette committed:
Symphony Self Registration

What I have so far for Symphony self reg forms - I'm editing getSelfRegistrationFields() in SirsiDynixROA() to show the fields specified in whatever self registration form is selected for the current library.
I moved a few settings from primary configuration to the self registration form (city/state vs city and state, SMS promting, and self registration profile)

mdnoble73 committed:
Merge pull request #1437 from catsoup11789/23.10.00

app system messages, aci changes

mdnoble73 committed:
Merge pull request #1438 from K-Alette/23.10.00

Symphony Self Registration

mdnoble73 committed:
Minor updates

mdnoble73 committed:
Wording updates

mdnoble73 committed:
Fix database maintenance message

mdnoble73 committed:
Backwards compatibility with PHP 7

mdnoble73 committed:
When updating home library, also update preferred pickup location if the home library does not allow preferred pickup location to be updated or remembered

catsoup11789 committed:
fix spelling

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

mdnoble73 committed:
Symphony Self Reg updates

mrenvoize commented on code/web/interface/themes/responsive/css-rtl/results-list.less:
```diff
@@ -20,6 +20,11 @@
   font-weight: bold;
   color: #44484a;
 }
+.fa-angle-down {
```
I'm surprised there's a need for this.. do we now user other font awesome assets in a similar way? I don't see targetted styles for those cases?

Less code is easier to maintain code ;P

mdnoble73 committed:
Update SirsiDynixROA.php

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

K-Alette committed:
Update 23.10.00.MD

catsoup11789 committed:
Update 23.10.00.MD

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

K-Alette committed:
Update 23.10.00.MD

mdnoble73 committed:
Merge pull request #1439 from K-Alette/23.10.00

Update 23.10.00.MD

catsoup11789 committed:
Update 23.10.00.MD

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

catsoup11789 committed:
Update 23.10.00.MD

mdnoble73 committed:
Merge pull request #1440 from catsoup11789/23.10.00

release notes, spelling fix

mdnoble73 committed:
Update 23.10.00.MD

catsoup11789 committed:
Update package.json

catsoup11789 committed:
Update yarn.lock

mdnoble73 committed:
Don't show empty user messages

mdnoble73 committed:
Update sierra_export_api.jar

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Rebuild sierra_export_api.jar

mdnoble73 committed:
Update IndexingProfile.php

mdnoble73 committed:
Fix to not show blank System messages

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

mdnoble73 committed:
Fix submitting quick polls if users are not logged in.

mdnoble73 committed:
Merge branch '23.09.03' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

dependabot[bot] committed:
Bump postcss from 8.4.19 to 8.4.31 in /code/aspen_app

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.19 to 8.4.31.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.19...8.4.31)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

mdnoble73 committed:
Merge pull request #1441 from mdnoble73/dependabot/npm_and_yarn/code/aspen_app/postcss-8.4.31

Bump postcss from 8.4.19 to 8.4.31 in /code/aspen_app

mdnoble73 committed:
Polaris Debugging

catsoup11789 committed:
add codabar to self-check scanner

catsoup11789 committed:
Merge branch '23.10.00' of https://github.com/catsoup11789/aspen-discovery into 23.10.00

Commit:
A new variable and a new conditional have been added
- New variable saves data about solr host
- The condition checks solr host while checkBackgroundProcesses is running

mdnoble73 committed:
Misc updates

- Increase the length of locationOwnedScopes, libraryOwnedScopes, recordIncludedScopes
- Add checking if a library has no hours when generating the open hours message.

catsoup11789 committed:
fix for accessing protected elements

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

mdnoble73 committed:
Merge pull request #1443 from catsoup11789/23.10.00

Updates, fixes

mdnoble73 committed:
Merge pull request #1442 from thekesolutions/solr_host_variable

A new variable and a new condition, related to Solr, have been added

mdnoble73 committed:
Add default solrHost

catsoup11789 committed:
check if copy is holdable

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

K-Alette committed:
Symphony Self Reg

moved CITY/STATE vs City AND State variable from self registration form and back into primary config
reorganized selfRegister() in SirsiDynixROA.php to check for custom selfReg forms and loop through each field in order of weight

mdnoble73 committed:
Remove unused database cleanup for expired Sessions

K-Alette committed:
Update SirsiDynixROA.php

make CITY/STATE field into two fields if using that instead of CITY and STATE

mdnoble73 committed:
Merge pull request #1446 from catsoup11789/23.10.00

check if copy is holdable for getCopies API

mdnoble73 committed:
Fix error running Database Update

mdnoble73 committed:
Include class

mdnoble73 committed:
Merge pull request #1445 from K-Alette/23.10.00

Symphony Self Reg

mdnoble73 committed:
Fix typo

mdnoble73 committed:
Merge branch '23.10.00' of https://github.com/mdnoble73/aspen-discovery into 23.10.00

mdnoble73 committed:
Additional error trapping and logging when loading MARC holdings

mdnoble73 committed:
Correct issues loading very large reading histories

mdnoble73 committed:
Allow patron usernames to be created and changed within Sierra

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Rebuild jars

catsoup11789 committed:
check treatBibOrItemHoldsAs for holdType

mdnoble73 committed:
Wait for Marc Holdings update thread to finish

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Update sierra_export_api.jar

mdnoble73 committed:
Merge pull request #1447 from catsoup11789/23.10.00

check treatBibOrItemHoldsAs for holdType

catsoup11789 committed:
Update 23.10.00.MD

catsoup11789 committed:
Update 23.10.00.MD

catsoup11789 committed:
Merge branch 'mdnoble73:23.10.00' into 23.10.00

mdnoble73 committed:
Merge pull request #1448 from catsoup11789/23.10.00

Update 23.10.00.MD

mdnoble73 committed:
Ignore spaces and hyphens within last names when validating patrons using barcode and last name

mdnoble73 committed:
Merge branch '23.10.00' of https://github.com/mdnoble73/aspen-discovery into 23.10.00

mdnoble73 committed:
Fix warning when user does not exist in Koha

mdnoble73 committed:
Send the correct message after updating username in Sierra

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
If a library has no local MARC Holdings do not collapse other MARC Holdings automatically and remove the label.

mdnoble73 committed:
Fix error validating users when user does not exist in Koha

mdnoble73 committed:
Merge branch '23.09.03' into 23.09.10

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Additional checking for records without Grouped works (while loading reading history)

mdnoble73 committed:
Merge branch '23.09.10' into 23.10.00

mdnoble73 committed:
Create starting release notes

mdnoble73 committed:
Create 23.11.00.MD
